README.md

```diff
@@ -1 +1,198 @@
-# Resort_Reserve_Front-end
\ No newline at end of file
+<a name="readme-top"></a>
+<h1 align='center'>Final Capstone - Resort Booking</h1>
+
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [ Rails Blog ](#-My-Blog-App-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+    - [🚀 Live Demo](#live-demo)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Database](#database)
+    - [Usage](#usage)
+    - [Testing](#testing)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+
+# Rails Blog <a name="about-project"></a>
+> Resort booking is a full-stack webapp which allows a user to manage their bookings. This is the front-end part of the project, if you want to see the back-end part, you can click [here](https://github.com/Thinus01/Resort_Booking_Back-end)
+
+## 🛠 Built With <a name="built-with"></a>
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Ruby On Rails with React</summary>
+  <ul>
+    <li><a href="">Postgresql</a></li>
+  </ul>
+</details>
+
+
+### Key Features <a name="key-features"></a>
+
+- Create/remove a booking for a resort
+- See a list of all possible resorts
+- Admins are able to add resorts to the list
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Visit our webpage here.
+
+- [Live Demo Link](https://resort-booking-front-end.onrender.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Mac or PC
+- Understanding OOP principles
+- Have the following installed on your machine
+    - Ruby
+    - Rails
+    - PostgreSQL
+    - React
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd your-folder
+  https://github.com/Thinus01/Resort_Booking_Front-end
+```
+
+### Install
+
+Install this project with:
+
+Install the required gems with:
+
+```sh
+bundle install
+npm install
+```
+
+### Database
+
+Create the databases and run migrations with:
+
+```sh
+rails db:create
+rails db:migrate
+```
+
+### Usage
+
+To run the development server, execute the following command:
+
+```sh
+rails server
+```
+
+### Testing
+
+To run tests, run the following command:
+
+```sh
+rspec spec 
+```
```

Comment:
- [ ] All these instructions pertain to the backend project.  Kindly, remove them and add specific instructions about how to setup and run the _frontend_ project.